Ports/.port_include.sh

```diff
@@ -77,9 +80,34 @@ run() {
     echo "+ $@"
     (cd "$workdir" && "$@")
 }
-run_replace_in_file(){
+run_replace_in_file() {
     run perl -p -i -e "$1" $2
 }
+install_launcher() {
+    if [ -z "$launcher_name" ] || [ -z "${launcher_category}" ] || [ -z "${launcher_command}" ]; then
+        return
+    fi
+    script_name="${launcher_name,,}"
+    script_name="${script_name// /}"
+    mkdir -p $DESTDIR/usr/local/libexec
+    cat >$DESTDIR/usr/local/libexec/$script_name <<SCRIPT
+#!/bin/sh
+set -e
+cd "\$(dirname -- \$(which -- $(printf %q "${launcher_command%% *}")))"
```

Comment:
you forgot to add quotes around the which expansion:
```suggestion
cd "\$(dirname -- "\$(which -- $(printf %q "${launcher_command%% *}"))")"
```

Reply:
Not sure what I'd do without such a thorough review. Thanks! Updated again.

Reply:
also, why did you remove the `--` after `cd`? POSIX requires that `cd` supports

Reply:
`exec` doesn't have/need it because it doesn't have options, but `cd` needs it.

Reply:
ping @gunnarbeutner